swagger.yaml

```diff
@@ -133,6 +133,115 @@ paths:
       security:
         - locations_api_key: []
 
+  /locations/{locationID}/services:
+    get:
+      tags:
+        - locations
+        - services
+      description: "Lists services related to a location"
+      operationId: getServices
+      produces:
+        - application/json
+      parameters:
+        - name: locationID
+          in: path
+          description: "ID of location"
+          required: true
+          type: string
+      responses:
+        "200":
+          description: "Successful response"
+          schema:
+            $ref: "#/definitions/ServicesResultObjects"
+        "404":
+          description: "Resource not found"
+          schema:
+            $ref: "#/definitions/Errors"
+        "500":
+          description: "Internal Server Error"
+          schema:
+            $ref: "#/definitions/Errors"
+      security:
+        - locations_api_key: []
+
+  /services:
+    get:
+      tags:
+       - services
+      # Describe this verb here. Note: you can use markdown
+      summary: Get services
+      description: "Gets all `Service` objects. Can be filtered by parameters."
+      operationId: getServicesList
+      produces:
+        - application/json
+      parameters:
+        - in: query
+          name: q
+          description: "Query to search through buildings. Looks through the building name, abbreviation."
+          required: false
+          type: string
+        - in: query
+          name: isOpen
+          description: "True if only search for open restaurants/buildings. Default: false to search for all the results"
+          required: false
+          type: boolean
+        - in: query
+          name: page[size]
+          description: "Number of results to return. Used in pagination. Default value is 10."
+          required: false
+          type: string
+        - in: query
+          name: page[number]
+          description: "Page number of results. Used to paginate through results."
+          required: false
+          type: string
+        - $ref: '#/parameters/pretty'
+      responses:
+        "200":
+          description: "Successful response"
+          schema:
+            $ref: "#/definitions/ServicesResultObjects"
+        "404":
```

Comment:
Is this applicable here? If a /services query returned nothing, it will still return a 200 with an empty data array.